### PR TITLE
Add plugins to split rest api by tags

### PR DESCRIPTION
**Pull request by hellosnow**

### CODE REVIEW

**Comment by vwxyzh** [comment]
ignore assembly `Microsoft.DocAsCode.Build.RestApi` for plugin loader?

**Comment by hellosnow** [comment]
Checked, test project references rest api project while plugins only references datacontract, so keep it is.

---
In reply to: [312978580](https://github.com/dotnet/docfx/pull/1827#issuecomment-312978580) [](ancestors = 312978580)

File: test/Microsoft.DocAsCode.Build.RestApi.WithPlugins.Tests/SplitRestApiToTagsLevelTest.cs [hunk 1]
```diff
@@ -9,6 +9,10 @@ namespace Microsoft.DocAsCode.Build.RestApi.Tests
 
     using Microsoft.DocAsCode.Build.Engine;
     using Microsoft.DocAsCode.Build.TableOfContents;
+    using Microsoft.DocAsCode.Build.TagsLevelRestApi;
```
Comment (vicancy) [inline test/Microsoft.DocAsCode.Build.RestApi.WithPlugins.Tests/SplitRestApiToTagsLevelTest.cs]: >TagsLevel [](start = 36, length = 9)

naming: TagLevel?

**Comment by vicancy** [comment]
:shipit:

**Pull request merged**